addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -8,14 +8,42 @@
     .o_view_nocontent {
         top: 10%;
 
-        .o_view_nocontent_expense_receipt:before {
+        .o_view_nocontent_expense_receipt_image {
             @extend %o-nocontent-init-image;
             width: 300px;
             height: 230px;
             background: transparent url(/hr_expense/static/img/nocontent.png) no-repeat center;
             background-size: 300px 230px;
-            margin-bottom: 0.75rem;
+            margin: 0.65rem auto;
         }
+
+        @include media-breakpoint-down(md) {
+            position: fixed;
+        }
+    }
+
+    .o_view_pink_overlay {
+        border: 2px;
+        border-style: dashed;
+        border-radius: 10px;
+        border-color: pink;
+        background-color: #f2edf0;
+        width: 600px;
+        height: 300px;
```

Comment:
Hello, we've been working this weekend with rugo to introduce a short animation when clicking on the "try a sample receipt" which should work in white mode and dark mode (task-3946192 & task-394615). While doing so I noticed that the pink overlay you introduced provides an unexpected visual result in darkmode.

![image](https://github.com/odoo/odoo/assets/118886338/24172220-9120-480a-b351-102f22cb610a)


I did a commit https://github.com/odoo/odoo/commit/a8de9376ee502b8e7e1a4237e2e7bd573cf57504 addressing this issue to merge with @rugo-odoo's work but since your PR is still under review it would be nice if you could adapt it here directly. This commit makes use of variables to reduce the styling while maintaining consistency between modules and making it work in darkmode. Feel free to add these change in your PR (the commit is from the branch `master-imp-expense_sample_receipt_animation-rugo-mano` but feel free to adapt it).